website/addons/wiki/model.py

```diff
@@ -175,13 +176,18 @@ class NodeWikiPage(GuidStoredObject, Commentable):
     user = fields.ForeignField('user')
     node = fields.ForeignField('node')
 
+    # For Django compatibility
+    @property
+    def pk(self):
+        return self._id
+
     @property
     def deep_url(self):
-        return '{}wiki/{}/'.format(self.node.deep_url, self.page_name)
+        return '{}wiki/{}/'.format(self.node.deep_url, urllib.quote(self.page_name))
```

Comment:
This fixes a bug with how the url is constructed. If there's a space in the wiki page name, there will be a space in the url since it's not being encoded. Originally, I fixed it for just one of the wiki url property methods so I changed it for these as well.